src/osemgrep/cli_ci/Github_metadata.ml

```diff
@@ -202,15 +202,27 @@ let _find_branchoff_point_from_github_api env =
   | Some gh_token, Some api_url, Some head_branch_hash -> (
       let headers = [ ("Authorization", Fmt.str "Bearer %s" gh_token) ] in
       let open Lwt.Infix in
-      Http_lwt_client.request ~meth:`GET ~headers
+        (* This is the worst thing I've ever done, I am so sorry
```

Comment:
No!
Fix it.
The code was working before, just define your own http_helpers.ml functions to not bother
existing code.

Reply:
Oh my bad 😭 I left this as a placeholder when I was moving between libraries, will fix...